clang/runtime/dpct-rt/include/dpct/dpl_extras/dpcpp_extensions.h

```diff
@@ -566,6 +566,49 @@ class radix_sort {
   uint8_t *_local_memory;
 };
 
+/// Load linear segment items into block format across threads
+/// Helper for Block Load
+template <size_t GROUP_WORK_ITEMS, typename InputT, int ITEMS_PER_THREAD,
```

Comment:
Do we need a signed `int` for `ITEMS_PER_THREAD`?

Reply:
Yes updated

Reply:
I think you maybe misunderstood my comment.  I'm suggesting that we will only have positive numbers for `ITEMS_PER_THREAD`, and so we do not need `int` (which is signed) for it or for `item`, but rather we could use `uint32_t`, `uint16_t` or something like that which is unsigned.

Reply:
Yes it was an error on my part.